.github/workflows/main.yml

```diff
@@ -39,24 +39,44 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
 
+      - name: Export environment variables
+        run: |
+          echo "SECRET_KEY=b6451545d2e4635148d768f07877aade3ad8e7e160d52962badd7587a4b9a150" > .env
+          echo "SMTP_HOST=smtp.gmail.com" >> .env
+          echo "SMTP_PORT=465" >> .env
+          echo "EMAIL_SENDER=test@kernelci.org" >> .env
+          echo "EMAIL_PASSWORD=random" >> .env
+
+      - name: Build docker images
+        run: docker-compose -f test-docker-compose.yaml build
+
```

Comment:
Moving unit tests to `test.yml` would also remove duplication of this steps.

Reply:
Then `pylint` step can also be moved to `test.yml` or maybe create a separate workflow file for linter tools?
Also, using the same container as unit test for linter doesn't look good to me.